bundled/README.md

```diff
@@ -19,6 +19,15 @@ and licensed under the Boost Software License Version 1.0. See
 A full version of the library can be downloaded at http://www.boost.org/.
 
 
+cpp-taskflow-*
```

Comment:
should we use ``cpp-taskflow-v2.4.0`` for the folder?

Reply:
yes, we should. I will fix that.